src/Hazelcast.Net/Clustering/MemberConnectionQueue.cs

```diff
@@ -28,6 +28,7 @@ namespace Hazelcast.Clustering
     internal class MemberConnectionQueue : IAsyncEnumerable<MemberConnectionRequest>, IAsyncDisposable
```

Comment:
Are the changes here only backports from the PR https://github.com/hazelcast/hazelcast-csharp-client/pull/635/files?show-viewed-files=true&file-filters%5B%5D= ?